firefox-ios/Client/Frontend/Settings/ContentBlockerSettingViewController.swift

```diff
@@ -145,20 +149,19 @@ class ContentBlockerSettingViewController: SettingsTableViewController {
         guard let defaultFooter = _defaultFooter else { return nil }
 
         if section == 0 {
-            // TODO: Get a dedicated string for this.
-            let title: String = .TrackerProtectionLearnMore
+            let linkButtonViewModel = LinkButtonViewModel(
+                title: .TrackerProtectionLearnMore,
+                a11yIdentifier: AccessibilityIdentifiers.Settings.ContentBlocker.title,
+                font: FXFontStyles.Regular.caption1.scaledFont()
+            )
+            linkButton.configure(viewModel: linkButtonViewModel)
 
-            let theme = themeManager.currentTheme(for: windowUUID)
-            let font = FXFontStyles.Regular.caption1.scaledFont()
-            var attributes = [NSAttributedString.Key: AnyObject]()
-            attributes[NSAttributedString.Key.foregroundColor] = theme.colors.actionPrimary
-            attributes[NSAttributedString.Key.font] = font
+            linkButton.applyTheme(theme: currentTheme)
```

Comment:
if we're applying the theme in `applyTheme` for this class, we can remove this here, which can remove creating the `currentTheme` at the class level. Instead we can create it in the local scope of the `applyTheme` method. Let me know if there's any issues with this.